### PR TITLE
UI: add merge strategy

### LINKED ISSUE #3266: Feature: Add Merge Strategy to the UI
Upon conflict, it would be great if we could select "accept source" or "accept dest".

### PR DESCRIPTION
**Pull request by eden-ohana**
closes #3266

### CODE REVIEW

**Comment by itaiad200** [comment]
Added @johnnyaug + @ozkatz for product feedback on the text

File: webui/src/pages/repositories/repository/compare.jsx [hunk 1]
```diff
@@ -159,37 +157,96 @@ const CompareList = ({ repo, reference, compareReference, prefix, onSelectRef, o
...
+                    <Modal.Title>Merge branch {source} into {dest}</Modal.Title>
+                </Modal.Header>
+                <Modal.Body>
+                    <FormControl>
+                        <FormLabel id="demo-radio-buttons-group-label">Merge Strategy</FormLabel>
+                        <FormHelperText>
+                            In case of a merge conflict, this option will force the merge process
+                            to automatically favor changes from the dest branch (&rdquo;dest-wins&rdquo;) or
+                            from the source branch(&rdquo;source-wins&rdquo;). In case no selection is made,
+                            the merge process will fail in case of a conflict.
```
[the remark this answers is not in the document]
Comment (eden-ohana) [reply webui/src/pages/repositories/repository/compare.jsx]: done

File: webui/src/pages/repositories/repository/compare.jsx [hunk 2]
```diff
@@ -159,37 +157,96 @@ const CompareList = ({ repo, reference, compareReference, prefix, onSelectRef, o
...
+                        </FormHelperText>
+                        <RadioGroup
+                            aria-labelledby="demo-radio-buttons-group-label" defaultValue="none" name="radio-buttons-group" row>
+                            <FormControlLabel value="none" control={<Radio />} label="none" onChange={onStrategyChange}/>
```
[the remark this answers is not in the document]
Comment (eden-ohana) [reply webui/src/pages/repositories/repository/compare.jsx]: done

**Review by johnnyaug — APPROVED** [review]
Yay

**Pull request merged**